zarr/util.py

```diff
@@ -365,3 +367,142 @@ def __repr__(self):
     def _repr_html_(self):
         items = self.obj.info_items()
         return info_html_report(items)
+
+
+class ZarrGroupTraversal(Traversal):
+
+    def get_children(self, node):
+        return getattr(node, "values", lambda: [])()
+
+    def get_root(self, tree):
+        return tree
+
+    def get_text(self, node):
+        name = node.name.split("/")[-1] or "/"
+        name += "[...]" if hasattr(node, "dtype") else ""
+        return name
+
+
+def custom_html_sublist(group, indent):
+    traverser = ZarrGroupTraversal(tree=group)
+    result = ""
+
+    result += (
+        """{0}<li><div>{1}</div>""".format(
+            indent, traverser.get_text(group)
+        )
+    )
+
+    children = traverser.get_children(group)
+    if children:
+        result += """\n{0}{0}<ul>\n""".format(indent)
+    for c in children:
+        for l in custom_html_sublist(c, indent).splitlines():
+            result += "{0}{0}{1}\n".format(indent, l)
+    if children:
+        result += "{0}{0}</ul>\n{0}".format(indent)
+
+    result += (
+        """</li>\n""".format(
+            indent, traverser.get_text(group)
```

Comment:
No format placeholders in this string, did you mean to call .format() on this?

Reply:
Good catch! That's likely a remnant from code changes. Will clear that out.